### ISSUE TITLE
@types/react support new <img> loading attribute

### ISSUE THREAD

**Issue by sami616** [DefinitelyTyped/DefinitelyTyped#38755]
Add support for the new loading image attribute.

**loading?: "eager" | "lazy" | "auto"**

```typescript
    interface ImgHTMLAttributes<T> extends HTMLAttributes<T> {
        alt?: string;
        loading?: "eager" | "lazy" | "auto"
        crossOrigin?: "anonymous" | "use-credentials" | "";
        decoding?: "async" | "auto" | "sync";
        height?: number | string;
        sizes?: string;
        src?: string;
        srcSet?: string;
        useMap?: string;
        width?: number | string;
    }
```

**Comment by eps1lon** [comment]
We don't include non-standard attributes. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38694#pullrequestreview-294671505 (and follow linked content) for more information.

**Comment by sami616** [comment]
okay thanks.

**Comment by kachkaev** [comment]
This feature should be hopefully merged in #38390 given a recent attribute acceptance.
